.github/workflows/pr-title-check.yml

```
@@ -0,0 +1,68 @@
+
+jobs:
+  check-title:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check PR title follows conventional commits
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const title = context.payload.pull_request.title;
+            const validPrefixes = ['feat:', 'fix:', 'docs:', 'chore:', 'refactor:', 'test:', 'ci:', 'perf:', 'style:'];
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)